evo-delivery-app/evo-delivery-backend/src/types/order.type.ts

```diff
@@ -0,0 +1,8 @@
+import { Address } from './address.type';
+
+export type Order = {
+  id: string;
+  shippingAddress: Address;
+  shippingDate: Date;
+  weight: number;
```

Comment:
Let's add 'createdAt' field so the user will have more details about the entities